components/turnstile.tsx

```
@@ -24,109 +24,113 @@ declare global {
 
-  const renderWidget = useCallback(() => {
-    if (!containerRef.current || !window.turnstile) return;
+    const renderWidget = useCallback(() => {
+      if (!containerRef.current || !window.turnstile) return;
 
-    const size =
-      window && window.matchMedia("(max-width: 440px)").matches
-        ? "compact"
-        : "flexible";
+      const size =
+        window && window.matchMedia("(max-width: 440px)").matches
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)